src/main/java/org/dice_group/grp/grammar/digram/DigramHelper.java

```diff
@@ -70,6 +89,77 @@
 		// TODO Auto-generated method stub
 		return null;
 	}
+	
+	/**
+	 * Finds the digram occurrences revolving around a specific statement in a given model
+	 * @param graph
+	 * @param newStmt
+	 * @return
+	 */
+	public static Set<DigramOccurence> findStmtBasedDigrams(Model graph, Statement newStmt) {
+		Set<DigramOccurence> occurrences = new HashSet<DigramOccurence>();
+		Resource n1 = newStmt.getSubject();
+		Property e1 = newStmt.getPredicate();
+		RDFNode n2 = newStmt.getObject();
+		
+		StringBuilder spec_case_1 = new StringBuilder("select * where { ");
```

Comment:
please stick to camelCase :)

Reply:
sorry about that, got the naming convention confused with python

Reply:
no worries, happens. 
If i script a lot in python i tend to forget ; :D